Doc/library/subprocess.rst

```diff
@@ -337,8 +337,8 @@ functions.
                  stderr=None, preexec_fn=None, close_fds=True, shell=False, \
                  cwd=None, env=None, universal_newlines=None, \
                  startupinfo=None, creationflags=0, restore_signals=True, \
-                 start_new_session=False, pass_fds=(), *, \
-                 encoding=None, errors=None, text=None)
+                 user=None, groups=None, start_new_session=False, \
```

Comment:
`user` and `groups` need to be added as keyword-only arguments.

Reply:
Done